upload/upload.go

```diff
@@ -109,23 +116,25 @@ func NewHandler(
 
 		observeSize(userAgent, size)
 
-		requestLogger = requestLogger.WithFields(logrus.Fields{"content-type": contentType, "size": size})
+		requestLogger = requestLogger.WithFields(logrus.Fields{"content-type": contentType, "size": size, "request_id": reqID, "account": id.Identity.AccountNumber})
```

Comment:
Can we also add `"orgid": id.Identity.Internal.OrgID` like we have below?

Reply:
Done!